src/klein/test/test_resource.py

```diff
@@ -1153,3 +1155,12 @@ def test_afUnixSocket(self):
         self.assertIsInstance(server_port, int)
         self.assertIsInstance(path_info, unicode)
         self.assertIsInstance(script_name, unicode)
+
+
+if _PY3:
+    from sys import version_info as pyVersion
+
+    if pyVersion.major > 3 or (pyVersion.major == 3 and pyVersion.minor >= 5):
```

Comment:
For the most part, Py2 and Py3 syntax shouldn't be too different, so do we really need a separate file for Py3? If it's just housing the ``async`` syntax stuff then this is fine, but with an all-encompassing name like ``py3_test_resource.py``, I'm inclined to think all Py3 tests would go in there.